test_runner/regress/test_compute_migrations.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import annotations
+
+from pathlib import Path
+from typing import TYPE_CHECKING, cast
+
+import pytest
+from fixtures.compute_migrations import NUM_COMPUTE_MIGRATIONS
+
+if TYPE_CHECKING:
+    from fixtures.neon_fixtures import NeonEnv
+
+
+MIGRATIONS: tuple[str, ...] = (
+    "0001-neon_superuser_bypass_rls",
+    "0002-alter_roles",
+    "0003-grant_pg_create_subscription_to_neon_superuser",
+    "0004-grant_pg_monitor_to_neon_superuser",
+    "0005-grant_all_on_tables_to_neon_superuser",
+    "0006-grant_all_on_sequences_to_neon_superuser",
+    "0007-grant_all_on_tables_to_neon_superuser_with_grant_option",
+    "0008-grant_all_on_sequences_to_neon_superuser_with_grant_option",
+    "0009-revoke_replication_for_previously_allowed_roles",
+    "0010-grant_snapshot_synchronization_funcs_to_neon_superuser",
+    "0011-grant_pg_show_replication_origin_status_to_neon_superuser",
+)
+
+
+def test_migrations_smoke(neon_simple_env: NeonEnv, compute_migrations_dir: Path):
+    """
+    Test that migrations ran and succeeded, or were skipped.
```

Comment:
Maybe `test_migrations_retry`? Because this seems to be the meaning of the test -- that we can safely retry after failure